src/components/pulse-dashboard/DashboardOverviewCharts.tsx

```
@@ -401,327 +518,645 @@ export function DashboardOverviewCharts({ currentRun, loading, error }: Dashboar
+              </div>
+              <div className="text-center p-4 bg-muted/50 rounded-lg">
+                <div className="text-3xl font-bold text-foreground">
+                  {retryStats.maxRetries}
+                </div>
+                <p className="text-sm text-muted-foreground mt-2 font-medium">
+                  Max Retries
+                </p>
+                {retryStats.maxRetryTests.length > 0 && (
+                  <Popover>
+                    <PopoverTrigger asChild>
+                      <button className="text-xs text-muted-foreground mt-2 truncate cursor-pointer border-b border-dotted border-muted-foreground/50 inline-block max-w-[180px] hover:text-foreground transition-colors focus:outline-none">
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)